src/MSALWrapper/Constants.cs

```diff
@@ -26,5 +26,10 @@ public static class Constants
         /// The aad redirect uri.
         /// </summary>
         public static readonly Uri AadRedirectUri = new Uri("http://localhost");
+
+        /// <summary>
+        /// The name of an environment variable used to disable file cache configuration.
+        /// </summary>
+        internal const string OEAUTH_MSAL_DISABLE_CACHE = "OEAUTH_MSAL_DISABLE_CACHE";

```

Comment:
Not a blocker (and in fact it would be a breaking change if we made it), but this should probably be `AZUREAUTH_MSAL_DISABLE_CACHE` to match our current environment variable naming convention.

Reply:
Good catch!